rest-api/dao/participant_dao.py

```diff
@@ -185,6 +255,9 @@ def to_client_json(self, model):
         'suspensionStatus': model.suspensionStatus,
         'suspensionTime': model.suspensionTime
     }
+    format_json_hpo(client_json, self.hpo_dao, 'awardee'),
```

Comment:
So it seems like this might need to run once for hpoId and then do client_json['awardee'] = client_json['hpoId'].